<?xml version="1.0" encoding="UTF-8"?> 
<launch>
  <node name="tf_to_path" pkg="tf_to_trajectory" type="posestamped_to_path.py">
    <param name="frame_id" value="map"/>
    <param name="topic_name" value="/mavros/vision_pose/pose"/>
    <param name="out_topic_name" value="/my_path"/>
    <param name="append_rate" value="25"/>
  </node>
</launch>
